bin/dockerd-wrapper

```diff
@@ -48,4 +57,6 @@ set -- \
 
 workaround_lp1626019
 
-exec docker daemon "$@"
+workaround_lp1606510
+
+exec dockerd -G $default_socket_group "$@"
```

Comment:
Will this `-G` error out if the `docker` group doesn't exist?  I don't recall Docker's behavior here.